src/org/omegat/gui/editor/mark/AltTranslationsMarker.java

```diff
@@ -0,0 +1,39 @@
+package org.omegat.gui.editor.mark;
+
+import org.omegat.core.Core;
+import org.omegat.core.data.SourceTextEntry;
+import org.omegat.util.gui.Styles;
+
+import javax.swing.text.Highlighter;
+import java.util.Collections;
+import java.util.List;
+
+public class AltTranslationsMarker extends AbstractMarker {
+
+    protected static final Highlighter.HighlightPainter PAINTER = new TransparentHighlightPainter(
+            Styles.EditorColor.COLOR_MARK_ALT_TRANSLATION.getColor(), 0.5F);
+
```

Comment:
Please register as plugin.

```suggestion
    public static void loadPlugins() {
        Core.registerMarkerClass(AltTranslationsMarker.class);
    }
    public static void unloadPlugins() {
    }
```

Reply:
You may want to add class path in `Plugins.properties` file.